### ISSUE TITLE
Presence Doesn't Shows Up.

### ISSUE THREAD

**Issue by dohyun854** [Pizzabelly/EasyRP#159]
My friend(someone I know to be exact) used this method to display a presence, however the presence doesn't show up. What is the problem?

https://media.discordapp.net/attachments/807878328081448980/807895246507343872/unknown.png

**Comment by lolzyes** [comment]
Yeah, I'm having the same problem and I have no idea on how to fix this.
